### PR TITLE
Create svedapete - worldcup.md

### PR DESCRIPTION
**Pull request by Svedapete**
### Description of Change

- a new .md file created

### Test results

- No need for a test, just new information provided
 
### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

File: svedapete - worldcup.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Teams
+BEL
```
Comment (Svedapete) [inline svedapete - worldcup.md:2]: not happy about this

**Review by Svedapete — COMMENTED** [review]
Euro cup

**Comment by BernardVillar** [comment]
Of course BELGIUM

**Review by mj580 — APPROVED** [review]
approved! :+1:

**Pull request merged**